spec/builders/subscriber_auth_email_builder_spec.rb

```diff
@@ -12,6 +12,13 @@
       )
     end
 
+    before do
+      allow(PublicUrls).to receive(:url_for)
+        .with(base_path: destination,
+              token: token)
```

Comment:
Nitpick: I think this would look neater moved onto line 17.

Reply:
Sure, I've amended that commit 👍.